sdp.c

```diff
@@ -22,69 +24,39 @@
 #include "debug.h"
 
 
-static su_home_t *home = NULL;
-
-
 /* SDP Initialization */
 int janus_sdp_init(void) {
-	home = su_home_new(sizeof(su_home_t));
```

Comment:
On 17/07/16 11:09, Lorenzo Miniero wrote:
> In sdp.c
> <https://github.com/meetecho/janus-gateway/pull/578#discussion_r71079222>:
>
>>  /* SDP Initialization */
>>  int janus_sdp_init(void) {
>> -	home = su_home_new(sizeof(su_home_t));
>
> You mean the |janus_sdp_init()| and |janus_sdp_deinit()|? I kept them as
> placeholders, in case something comes up that needs setup there. But
> yeah, they're pretty much useless right now.
>

Yeah. IMHO I'd get rid of them, unless you already foresee something 
that would need to be initialized.


-- 
Saúl Ibarra Corretgé
http://bettercallsaghul.com